Error: SSR requires `@loadable/babel-plugin`, please install it

When I run `gatsby build` I get the error 
```bash
WebpackError: Invariant Violation: loadable: SSR requires `@loadable/babel-plugin`, please install it
```
I'm on version `3.2.0` and the gatsby version is `3.7.1`.

The loadable component is 
```jsx
import Loadable from "@loadable/component"

const DistortedShader = Loadable(() => import("./DistortedShader"))

export {DistortedShader}
```

And then I'm importing it inside an `MDX` file

```jsx
import { DistortedShader } from "../../../demos"

<DistortedShader />
```

Not sure what's the problem though 😅

Just tried `3.3.0`, now I'm getting this error 😅

```
WebpackError: Cannot find module 'DIR\.cache\loadable-stats-build-javascript.json'
```

It's now missing from `.cache` instead of `public`

--- a/src/gatsby-node.js
+++ b/src/gatsby-node.js
@@ -6,7 +6,7 @@ const isWin = process.platform === "win32";
 
 const loadableStatsName = "loadable-stats-build-javascript.json";
 
-const statsPath = _path.default.join(process.cwd(), `/.cache/${loadableStatsName}`);
+const statsPath = path.join(process.cwd(), `/.cache/${loadableStatsName}`);
 
 export const onCreateWebpackConfig = ({ actions, stage }) => {
   if (

--- a/src/gatsby-node.js
+++ b/src/gatsby-node.js
@@ -4,9 +4,9 @@ import LoadablePlugin from "@loadable/webpack-plugin"
 
 const isWin = process.platform === "win32";
 
-const loadableStatsRelativePath = "../.cache/loadable-stats-build-javascript.json";
+const loadableStatsName = "loadable-stats-build-javascript.json";
 
-const statsPath = isWin ? loadableStatsRelativePath : _path.default.join(process.cwd(), loadableStatsRelativePath);
+const statsPath = _path.default.join(process.cwd(), `/.cache/${loadableStatsName}`);
 
 export const onCreateWebpackConfig = ({ actions, stage }) => {
   if (
@@ -19,6 +19,7 @@ export const onCreateWebpackConfig = ({ actions, stage }) => {
         new LoadablePlugin({
           filename: statsPath,
           writeToDisk: true,
+          outputAsset: !isWin
         }),
       ],
     })

--- a/src/gatsby-node.js
+++ b/src/gatsby-node.js
@@ -2,13 +2,11 @@ import { unlinkSync } from "fs"
 import path from "path"
 import LoadablePlugin from "@loadable/webpack-plugin"
 
-const loadableStatsName = "loadable-stats-build-javascript.json";
-const statsPath = path.join(
-  process.cwd(),
-  `/.cache/${loadableStatsName}`
-)
-
 const isWin = process.platform === "win32";
+
+const loadableStatsRelativePath = "../.cache/loadable-stats-build-javascript.json";
+
+const statsPath = isWin ? loadableStatsRelativePath : _path.default.join(process.cwd(), loadableStatsRelativePath);
 
 export const onCreateWebpackConfig = ({ actions, stage }) => {
   if (
@@ -19,7 +17,7 @@ export const onCreateWebpackConfig = ({ actions, stage }) => {
     actions.setWebpackConfig({
       plugins: [
         new LoadablePlugin({
-          filename: (isWin ? loadableStatsName : statsPath),
+          filename: statsPath,
           writeToDisk: true,
         }),
       ],

